tasks/push-rpm-data-to-pyxis/tests/mocks.sh

```diff
@@ -45,6 +51,34 @@ function upload_rpm_data() {
   fi
 }
 
+function upload_rpm_data_cyclonedx() {
+  echo Mock upload_rpm_data_cyclonedx called with: $*
+  echo $* >> "$(workspaces.data.path)/mock_upload_rpm_data_cyclonedx.txt"
+
+  if [[ "$*" != "--retry --image-id "*" --sbom-path "*".json --verbose" ]]
+  then
+    echo Error: Unexpected call
+    exit 1
+  fi
+
+  if [[ "$3" == myImageID1Failing ]]
+  then
+    echo "Simulating a failing RPM data push..."
+    return 1
+  fi
+
+  if [[ "$3" == myImageID?Parallel ]]
```

Comment:
Doesn't only the new test file use cyclonedx? I don't see parallel images in it

Reply:
I just copied the other mock function, but you're right, this is not needed - also another `if` for a failure. Fixed.